cstore_compression.c

```diff
@@ -0,0 +1,165 @@
+/*-------------------------------------------------------------------------
+ *
+ * cstore_compression.c
+ *
+ * This file contains compression/decompression functions definitions
+ * used in cstore_fdw.
+ *
+ * Copyright (c) 2016, Citus Data, Inc.
```

Comment:
That's more like a 8 byte placeholder. Even structure name is different. I am not sure that requires a copyright message.